code/script.js

```diff
@@ -1,51 +1,246 @@
+  chatbot.classList.remove("chatbot");
+  setTimeout(() => greetUser(), 500);
 }
 
-// Starts here
+ticketBtn.addEventListener("click", botPopUp);
+
+// Start of the bot
+// Greeting and Question 1
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
+  // here we call the function showMessage, that we declared earlier with the argument "..." for message, and the argument "bot" for sender.
+  showMessage("Welcome to the Cinema City TicketBot!", 'bot');
+  setTimeout(() => {
+    showMessage("To proceed, I need the name of the person who makes the reservation. Please, enter your name.", 'bot');
+  }, 500);
+  sendBtn.addEventListener('click', handleNameInput);
+};
+
+// Customer name pops up in chat on user's side.
+const handleNameInput = (event) => {
+  event.preventDefault();
+  customerName = inputField.value;
+
+  if (customerName === "") {
+    alert(`You need to write your name.`);
+  } else {
+    showMessage(customerName, "user");
+    clearInputField();
+    // Shows the next question after 1s by calling the next function.
+    setTimeout(() => ticketType(), 1000);
+  };
+};
+
+// Question 2
+// Bot replies back and ask if the ticket are for a adult or child.
+const ticketType = () => {
+  removeChildrenFunction();
+  showMessage(`Nice to meet you ${customerName}! Is the movie ticket for a adult, a student or a child (Under 7)?`, 'bot');
+  // Two buttons for age choices.
+  inputWrapper.innerHTML += `
+    <button id="adult" type="submit">Adult</button>
+    <button id="student" type="submit">Student</button>
+    <button id="child" type="submit">Child</button>
+  `;
+  // Variables that store references to the two added buttons. 
+  const adultBtn = document.getElementById('adult');
+  const studentBtn = document.getElementById('student');
+  const childBtn = document.getElementById('child');
+  //Eventlisteners for the buttons.
+  adultBtn.addEventListener('click', () => {
+    ticketOption = reservation.type[0];
+    handleTicketType();
+  });
+  studentBtn.addEventListener('click', () => {
+    ticketOption = reservation.type[1];
+    handleTicketType();
+  });
+  childBtn.addEventListener('click', () => {
+    ticketOption = reservation.type[2];
+    handleTicketType();
+  });
+
+  // Function that shows a message with the user's choice and calls the function for the next question.
+  const handleTicketType = () => {
+    if (ticketOption === "Adult") {
+      price = ticketPrices.adultPrice;
+    } else if (ticketOption === "Student") {
+      price = ticketPrices.studentPrice;
+    } else if (ticketOption === "Child") {
+      price = ticketPrices.childPrice;
+    };
+
+    showMessage(`It's for a ${ticketOption.toLowerCase()}.`, 'user');
```

Comment:
You're using handy methods that we've learned through the course (toLowerCase()), well done!